2022-03-11 23:40:49,977 INFO    MainThread:83583 [wandb_setup.py:_flush():75] Loading settings from /Users/aitorarronte/.config/wandb/settings
2022-03-11 23:40:49,978 INFO    MainThread:83583 [wandb_setup.py:_flush():75] Loading settings from /Users/aitorarronte/PycharmProjects/gnn-speech-emotion/wandb/settings
2022-03-11 23:40:49,978 INFO    MainThread:83583 [wandb_setup.py:_flush():75] Loading settings from environment variables: {}
2022-03-11 23:40:49,978 WARNING MainThread:83583 [wandb_setup.py:_flush():75] Could not save program above cwd: /Applications/PyCharm.app/Contents/helpers/pydev/pydevconsole.py
2022-03-11 23:40:49,978 INFO    MainThread:83583 [wandb_setup.py:_flush():75] Inferring run settings from compute environment: {'program_relpath': None, 'program': '/Applications/PyCharm.app/Contents/helpers/pydev/pydevconsole.py'}
2022-03-11 23:40:49,978 INFO    MainThread:83583 [wandb_init.py:_log_setup():405] Logging user logs to /Users/aitorarronte/PycharmProjects/gnn-speech-emotion/wandb/run-20220311_234049-dhuow1vz/logs/debug.log
2022-03-11 23:40:49,978 INFO    MainThread:83583 [wandb_init.py:_log_setup():406] Logging internal logs to /Users/aitorarronte/PycharmProjects/gnn-speech-emotion/wandb/run-20220311_234049-dhuow1vz/logs/debug-internal.log
2022-03-11 23:40:49,978 INFO    MainThread:83583 [wandb_init.py:init():439] calling init triggers
2022-03-11 23:40:49,979 INFO    MainThread:83583 [wandb_init.py:init():443] wandb.init called with sweep_config: {}
config: {}
2022-03-11 23:40:49,979 INFO    MainThread:83583 [wandb_init.py:init():492] starting backend
2022-03-11 23:40:49,979 INFO    MainThread:83583 [backend.py:_multiprocessing_setup():101] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2022-03-11 23:40:49,999 INFO    MainThread:83583 [backend.py:ensure_launched():219] starting backend process...
2022-03-11 23:40:50,007 INFO    MainThread:83583 [backend.py:ensure_launched():225] started backend process with pid: 83596
2022-03-11 23:40:50,012 INFO    MainThread:83583 [wandb_init.py:init():501] backend started and connected
2022-03-11 23:40:50,024 INFO    MainThread:83583 [wandb_init.py:init():565] updated telemetry
2022-03-11 23:40:50,054 INFO    MainThread:83583 [wandb_init.py:init():596] communicating run to backend with 30 second timeout
2022-03-11 23:40:51,349 INFO    MainThread:83583 [wandb_run.py:_on_init():1759] communicating current version
2022-03-11 23:40:51,659 INFO    MainThread:83583 [wandb_run.py:_on_init():1763] got version response 
2022-03-11 23:40:51,659 INFO    MainThread:83583 [wandb_init.py:init():625] starting run threads in backend
2022-03-11 23:40:51,726 INFO    MainThread:83583 [wandb_run.py:_console_start():1733] atexit reg
2022-03-11 23:40:51,727 INFO    MainThread:83583 [wandb_run.py:_redirect():1606] redirect: SettingsConsole.REDIRECT
2022-03-11 23:40:51,729 INFO    MainThread:83583 [wandb_run.py:_redirect():1611] Redirecting console.
2022-03-11 23:40:51,731 INFO    MainThread:83583 [wandb_run.py:_redirect():1667] Redirects installed.
2022-03-11 23:40:51,731 INFO    MainThread:83583 [wandb_init.py:init():664] run started, returning control to user process
